src/main/sensors/compass.c

```diff
@@ -75,12 +77,13 @@
 #define P0 1.0e2f        // value to initialize P(0) = diag([P0, P0, P0, P0]), typically in range: (1, 1000)
 
 #define CALIBRATION_WAIT_US (15 * 1000 * 1000)               // wait for movement to start and trigger the calibration routine in us
-#define GYRO_NORM_SQUARED_MIN sq(DEGREES_TO_RADIANS(450.0f)) // minimal value that has to be reached so that the calibration routine starts in (rad/sec)^2,
+#define GYRO_NORM_SQUARED_MIN sq(DEGREES_TO_RADIANS(350.0f)) // minimal value that has to be reached so that the calibration routine starts in (rad/sec)^2,
                                                              // a relatively high value so that the calibration routine is not triggered too early
 #define CALIBRATION_TIME_US (30 * 1000 * 1000)               // duration of the calibration phase in us
 
-static timeUs_t tCal = 0;
+static timeUs_t magCalEndTime = 0;
 static bool didMovementStart = false;
+static bool magCalProcessActive = false;
```

Comment:
It is also possible to use `magCalEndTime != 0` as in old code. Probability of it failing (hitting `micros() == 0`) is small enough.

Reply:
Yes, we could use old `micros() == 0` method, but this way is, perhaps, a bit more transparent, and neatly separates out the `logical` aspect from the timing aspect.

Reply:
yes, it better in this respect.

In ideal world, whole BF codebase would use consistent approach, including naming conventions(so it is obvious which flag is associated with given timer). But that is long way to go..

Reply:
Ah, indeed there is much to do :-)